pkg/backup/item_backupper_test.go

```diff
@@ -106,6 +106,63 @@ func TestBackupItemSkips(t *testing.T) {
 	}
 }
 
+func TestShouldBackup(t *testing.T) {
+	tests := []struct {
+		name          string
+		expected      bool
+		content       string
+		groupResource schema.GroupResource
+		expectedErr   bool
+	}{
+		{
+			name:          "Failed pods aren't backed up",
+			expected:      false,
+			content:       `{"apiVersion":"v1","kind":"Pod","metadata":{"namespace":"ns","name":"pod1"}, "status": {"phase": "Failed"}}`,
+			groupResource: schema.GroupResource{Group: "", Resource: "pods"},
+		},
+		{
+			name:          "Succeeded pods aren't backed up",
+			expected:      false,
+			content:       `{"apiVersion":"v1","kind":"Pod","metadata":{"namespace":"ns","name":"pod1"}, "status": {"phase": "Succeeded"}}`,
+			groupResource: schema.GroupResource{Group: "", Resource: "pods"},
+		},
+		{
+			name:          "Pending pods are backed up",
+			expected:      true,
+			content:       `{"apiVersion":"v1","kind":"Pod","metadata":{"namespace":"ns","name":"pod1"}, "status": {"phase": "Pending"}}`,
+			groupResource: schema.GroupResource{Group: "", Resource: "pods"},
+		},
+		{
+			name:          "Jobs without a completion time are backed up",
+			expected:      true,
+			content:       `{"apiVersion":"v1","kind":"Pod","metadata":{"namespace":"ns","name":"pod1"}}`,
+			groupResource: schema.GroupResource{Group: "batch", Resource: "jobs"},
+		},
+		{
+			name:          "Jobs with a completion time aren't backed up",
+			expected:      false,
+			content:       `{"apiVersion":"v1","kind":"Pod","metadata":{"namespace":"ns","name":"pod1"}, "status": {"completionTime": "bar"}}`,
+			groupResource: schema.GroupResource{Group: "batch", Resource: "jobs"},
+		},
+		{
+			name:          "Something not a pod or a job should be backed up.",
```

Comment:
you could add a `phase` = Completed and non-empty `completionTime` to the status here just to validate that non-pods/jobs with those fields are backed up.